Project-II-Calculator/calculator/src/components/ButtonComponents/ActionButton.js

```diff
@@ -1,2 +1,21 @@
 import React from 'react';
+import Button from './Button';
 import './Button.css';
+
+let buttonStyle = {
+  width: '75%',
```

Comment:
This is amazing, I'm impressed!
You definitely want to have a look at <a href="https://www.styled-components.com/">styled-components</a> library, it's really great. It's basically an enhanced version of what you are doing here.
Very well done making your component really reusable!